CONTRIBUTING.md

```diff
@@ -27,6 +27,26 @@ satisfy the above requirement.
 This project has adopted the [CNCF Code of
 Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 
+## Rust Guidelines
+
+- **Channel**: Code is built and tested using the **stable** channel of Rust, but documented and formatted with **nightly**
```

Comment:
Nit: we should probably ensure that rustdoc on stable at least doesn't choke completely (since `cargo doc` builds docs recursively for all dependencies).

Reply:
doc actually works on stable, but it doesn't have features to make it look like docs.rs (the cfg flag is necessary to see how the "this is only avaiable with feature X" show up)

closest we have is:
```
cargo doc --lib --workspace --features=derive,ws,oauth,jsonpatch,client,derive,runtime,admission,k8s-openapi/v1_22
```

Reply:
Yeah, I know. I just mean that we shouldn't let this comment let us get laxer here in the future.